planning/autoware_route_handler/src/route_handler.cpp

```diff
@@ -370,15 +379,24 @@ void RouteHandler::setRouteLanelets(const lanelet::ConstLanelets & path_lanelets
 
   route_lanelets_.clear();
   route_lanelets_.reserve(route_lanelets_id.size());
+  std::vector<RouteRtreeNode> rtree_nodes;
+  rtree_nodes.reserve(route_lanelets_id.size());
+  auto i = 0UL;
```

Comment:
I think it is better.

```suggestion
  size_t i = 0;
```